<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialogue</class>
 <widget class="QDialog" name="AddItemDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblProductName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProductQuantity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProductImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Image:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbQuantity">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConfirmAdd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm and Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
